internal/database/migration_test.go

```
@@ -127,6 +128,34 @@ func TestMigrationSystem(t *testing.T) {
 		}
 	})
 
+	t.Run("options table has direction column", func(t *testing.T) {
+		var count int
+		if err := db.QueryRow("SELECT COUNT(*) FROM pragma_table_info('options') WHERE name='direction'").Scan(&count); err != nil || count != 1 {
+			t.Fatalf("expected options.direction column, count=%d err=%v", count, err)
+		}
+	})
+
+	t.Run("option direction defaults to Short and permits opposite legs", func(t *testing.T) {
+		if _, err := db.Exec("INSERT INTO symbols (symbol) VALUES ('NASDAQ:SPREAD')"); err != nil {
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)